style.css

```diff
@@ -13,10 +14,338 @@ h1,h2,h3,h4,h5,h6{
+    width: 211px;
+}
+.container__hero{
+    padding-top: 100px;
+    margin-left: 184px;
+}
+.hero__inner{
+    display: flex;
+}
+.hero__sub{
+    width: 459px;
+    margin-right: 77px;
+}
+.hero_title{
+    font-size: 5rem;
+    line-height: 77px;
+    letter-spacing: -2.50105px;
+    color: #19191B;
+    padding-top: 92px;
+}
+.hero__text{
+    font-family:'Mulish', sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 30px;
+    letter-spacing: -0.625263px;
+    color: #696871;
+}
+.hero_btn{
+    margin-top: 50px;
+    
+    font-family:'DM Sans', sans-serif;
+    font-size: 17px;
+    line-height: 22px;
+    text-align: center;
+    letter-spacing: -0.531474px;
+    color: white;
+    background-color:#5454D4 ;
+    border: none;
+    padding: 19px 59px;
+    border-radius: 10px;
+}
+.group{
+    padding-top: 105px;
+    
+}
+.group__logo{
+    display: flex;
+    
+}
+.group_logo_first{
+    padding-left: 186px;
+}
+.group_logo_log{
+    padding-left: 120px;
+}
+.container__index{
+    padding-left: 186px;
+    padding-top: px;
+    margin: 0 auto;
+}
+.index__inner{
+    display: flex;
+}
+.index__logo{
+    padding-top:148px ;
+}
+.index__sub{
+    padding-left: 165px;
+    padding-top: 160px;
+}
+.index__title{
+    font-size: 50px;
+    line-height: 57px;
+    letter-spacing: -1.56316px;
+    color: #19191B;
+    padding-top: 102px;
+    padding-bottom: 25px;
+}
+.index__text{
+    font-size: 20px;
+    line-height: 30px;
+    letter-spacing: -0.625263px;
+    color: #696871;
+    width: 459px;
+}
+.index_link{
+    padding-top: 30px;
+    text-decoration: none;
+    font-family: 'DM Sans', sans-serif;
+    font-size: 17px;
+    line-height: 22px;
+    letter-spacing: -0.531474px;
+    color: #5454D4;
+}
+.index__sub.second{
+    padding-left: 0;
+    padding-right: 158px;
+    padding-top: 154px;
+}
+.index__logo_second{
+    padding-top: 83px;
+}
+.container__card{
+    padding-left: 180px;
+    padding-top: 123px;
+}
+.card__inner{
+    display: flex;
+    justify-content: space-around;
+    text-align: center;
+}
+
+.card_title{
+    font-size: 24px;
+    line-height: 36px;
+    text-align: center;
+    letter-spacing: -0.750316px;
+    color: #19191B;
+    text-align: center;
+    padding-top: 7px;
+    padding-bottom: 10px;
+}
+.card_about{
+    font-size: 16px;
+    line-height: 24px;
+    text-align: center;
+    letter-spacing: -0.500211px;
+    color: #696871;
+    text-align: center;
+
+}
+.card__first{
+    padding-right: 55px;
+}
+.card__first_thirrd{
+    margin-right: 175px;
+}
+.footer__container{
+    margin-left: 165px;
+    padding-top: 130px;
+
+}
+.footer__title{
+    display: flex;
+    justify-content: space-between;
+}
+.footer_btn{
+    margin-right: 183px;
+    padding-top: 50px;
+}
+.footer_name{
+    width: 523px;
+    font-size: 60px;
+    line-height: 70px;
+    letter-spacing: -1.87579px;
+    color: #19191B;
+}
+.first_foot_bnt{
+    color: white;
```

Comment:
you should write color in one style
if write color as HEX, write it

Reply:
done